### ISSUE TITLE
Default debugging broken on Mac in latest release (2.7.1373)

### ISSUE THREAD

**Issue by EricJizbaMSFT** [Azure/azure-functions-core-tools#1447]
Repro steps:
1. Create a new C# project in VS Code
1. Add an http trigger
1. Debug

NOTE: This only repros on Mac, not windows.

Old behavior where debugging worked:
```
Azure Functions Core Tools (2.7.1373 Commit hash: cd9bfca26f9c7fe06ce245f5bf69bc6486a685dd)
Function Runtime Version: 2.0.12507.0
Skipping 'AzureWebJobsStorage' because value is either empty or is null
```

New behavior where debugging fails:
```
Azure Functions Core Tools (2.7.1468 Commit hash: 6798997398cb262a46ec366a9be2374e23c19910)
Function Runtime Version: 2.0.12555.0
Windows Setting AzureWebJobsStorage = 
Setting unsupported .NET environemt variables (empty string) is not implemented.
```

_originally filed in https://github.com/microsoft/vscode-azurefunctions/issues/1385 by @v-dantch_

**Comment by ptoonen** [comment]
Noticed this too earlier. It's because [this](https://github.com/Azure/azure-functions-core-tools/blob/a25784e1d2a60a02de99bb7af22b90e1f172b8c8/src/Azure.Functions.Cli/NativeMethods/EnvironmentNativeMethods.cs) currently only works on Windows.

**Comment by ptoonen** [comment]
I did some tests and the code below seems to work, but that also makes it a completely different implementation of getenv which would go too far I guess. So I'd say either remove the default variable, or do a check if the string is empty when it concerns a non-Windows machine? But I'm not sure whether that allows for the situation described in https://github.com/Azure/azure-functions-core-tools/issues/1258.

`        [DllImport("libc")]
        public static extern int setenv(string name, string value);


        [DllImport("libc")]
        public static extern IntPtr getenv(string name);

        static void Main(string[] args)
        {
            string emptyEnv = nameof(emptyEnv);
            setenv("emptyEnv", "");
            Console.WriteLine(Marshal.PtrToStringAnsi(getenv(emptyEnv)) == ""); // true
        }`

**Comment by ahmelsayed** [comment]
I'm going to remove the exception on non-Windows machines and print a warning if you run in debug. 

There is no way to set an environment variable to empty string in a .NET process on Linux/Mac after it starts. This is unfortunate because in Azure the process environment is set before start, and all OSs allow for empty string env vars. .NET is the problem here and I don't know how the cli could fix it.

**Comment by ahmelsayed** [comment]
To make matters more confusing, as #1258 shows the function runtime itself behaves differently if the value is null vs empty string, but there is no way to set that through .NET.

**Comment by fabiocav** [comment]
We should indeed remove this for now. We can think about the proper approach once this is resolved.

**Comment by soninaren** [comment]
This is fixed and released in version 2.7.1480
https://github.com/Azure/azure-functions-core-tools/releases/tag/2.7.1480

**Comment by EricJizbaMSFT** [comment]
Yep - works for me now. Thanks!
